install/kubernetes/ingress-sni-zvpn.yaml

```diff
@@ -0,0 +1,364 @@
+# Example configuration using Envoy v2 bootstrap, custom envoy configuration with SNI and
+# sample 'zero vpn' port forwarding. This needs to be manually updated for new domains and services,
+# work in progress to automate it by having Pilot generate LDS v2 with SNI and capture the zvpn ports from
+# annotations.
+#
+# Setting up SNI in LDSv2:
+# https://www.envoyproxy.io/docs/envoy/latest/faq/sni.html#faq-how-to-setup-sni
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: envoy-config
+  namespace: istio-system
+data:
+  envoy.yaml: |
```

Comment:
Now that we imported go-control-plane, can we use the schematize version of this YAML?
We want to preserve backwards compatibility with the existing switches in the mesh config (turning on and off mixer, tracing, etc).

Reply:
Not sure what you mean by 'schematize version' - but this will be turned into a template, this is the test version to verify the config works.